generators/service-alert-notification/templates/swift/instrumentation.swift

```diff
@@ -1,9 +1,9 @@
 import LoggerAPI
+import CloudEnvironment
 import AlertNotifications
 
-var serviceCredentials: ServiceCredentials!
-
-func initializeServiceAlertNotification() throws {
+func initializeServiceAlertNotification(cloudEnv: CloudEnv) throws -> ServiceCredentials {
+    let serviceCredentials: ServiceCredentials
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment.

Reply:
done